zask/ext/zerorpc/__init__.py

```diff
@@ -47,10 +47,12 @@
 CONFIG_CUSTOME_HEADER_MIDDLEWARE = 'header'
 ACCESS_LOG_MIDDLEWARE = 'access_log'
 REQUEST_CHAIN_MIDDLEWARE = 'uuid'
+REQUEST_EVENT_MIDDLEWARE = 'event'
 DEFAULT_MIDDLEWARES = [
     CONFIG_CUSTOME_HEADER_MIDDLEWARE,
     REQUEST_CHAIN_MIDDLEWARE,
-    ACCESS_LOG_MIDDLEWARE
+    ACCESS_LOG_MIDDLEWARE,
+    REQUEST_EVENT_MIDDLEWARE
```

Comment:
I don't think so, I just ordered them according to their declaration order in the file itself.